book/graphs/other/bwg01.qmd

```diff
@@ -79,6 +77,15 @@ bp_annos <- function(bp, color, annos = 1) {
 }
 ```
 
+```{r include = FALSE}
+webr_code_labels <- c("setup")
+```
+
+{{< include ../../_utils/webr_no_include.qmd >}}
+
+## Output
+
+:::::::::::: panel-tabset
```

Comment:
hi @pawelru , the number of ":" varies from template to template, I was wondering why is that? what does the ":" do?

Reply:
These were added automatically by Quarto extension in VSCode. It remains fully functional but I agree - this is just confusing. I will try to look at this again to see if we can make it more consistent across articles but on the other hand I want to limit these auto-edits in the future.

Reply:
This continue to get auto-updates. When I change it back to a fewer `:` then Quarto ext change it back to more. I think I don't want to go and dig into that. It's not a big deal.